R/join_keys-c.R

```diff
@@ -28,13 +28,13 @@ c.join_keys <- function(...) {
     f = function(.x, .y) {
       assert_compatible_keys2(.x, .y)
       out <- utils::modifyList(.x, .y, keep.null = FALSE)
-      attr(out, "parents") <- .merge_parents(.x, .y)
+      parents(out) <- .merge_parents(.x, .y)
```

Comment:
Validation for acyciclic graphs

Reply:
Now parents is always a list (empty or not, but never NULL), so I believe you can remove `.merge_parents` and replace it with `modifyList(.x, .y)`

Reply:
Yup. done!